index.html

```diff
@@ -61,13 +57,62 @@ <h2 class="content-center">Featured</h2>
         </div>
       </section>
     </main>
+    <div id="product-control">
+      <div class="product-control__icon" id="add-product"></div>
+    </div>
+    <div id="modal">
+      <div class="modal-container">
+        <form action="submit" class="modal__form">
```

Comment:
I believe the `action` attribute for a form expects a URL: https://www.w3schools.com/tags/att_form_action.asp  
Consider removing this attribute.